web-console/4.0-designs/storage/storage.md

```diff
@@ -29,3 +29,48 @@
 ![form for adding and creating storage](img/create-storage-form.png)
 - Selecting `Create Storage` exposes fields for Storage Class, Name, Access Mode, Size, and the label selector checkbox.
 - Only access modes that are compatible should be available to select in this form.
+
+## Set Default Storage Class
+- Users with appropriate privileges can set a Default Storage Class.
+- THERE ARE TWO WAY USER CAN SET A DEFAULT STORAGE CLASS:
+
+#### OPTION-1: Kebab menu option:
+
+1. Login to OpenShift > Admin > Storage (menu) > Storage classes (sub-menu)
+2. User can see list of ‘Storage classes' [list view] & there will be a label that indicates `default` storage class
+3. Click on the kebab menu > select `Set as Default Storage Class`
+
+![Set a Default Storage Class using kebab menu](img/stroage-opt1-start-1.png)
+
+4. Pop-up opens with confirmation message.
+5. User clicks on 'Confirm' button to make the change
+
+![Popup menu with confirmation](img/stroage-opt1-popup-confirm-1.png)
```

Comment:
It looks like you are using the PF4 modal styling, which I think is fine since we are heading that way in OpenShift, but I think the Cancel button treatment should just be a link styling: https://www.patternfly.org/v4/documentation/react/components/modal/

Reply:
Sure, I will update. Thank you @lizsurette